migration/wcmp-upgrade-migration.php

```diff
@@ -60,6 +60,24 @@ protected function migrateSettings(array $map, ?array $newSettings, array $oldSe
         return $newSettings;
     }
 
+    /**
+     * @param array  $map
+     * @param string $settingName
+     * @param mixed  $newValue
+     *
+     * @return array
+     */
+    protected function replaceValue(array $map, string $settingName, $newValue): array
```

Comment:
Graag een uitleg in phpdoc waarom je dit doet en wat het doet.

Reply:
Done